enforcer-parent/enforcer/src/main/java/org/wso2/choreo/connect/enforcer/admin/AdminUtils.java

```diff
@@ -47,14 +48,28 @@ public class AdminUtils {
     public static APIInfo toAPIInfo(API api, List<SubscriptionInfo> subscriptionInfoList) {
         APIInfo apiInfo = new APIInfo();
         apiInfo.setSubscriptions(subscriptionInfoList);
-        apiInfo.setApiId(api.getApiId());
-        apiInfo.setApiUUID(api.getApiUUID());
-        apiInfo.setContext(api.getContext());
-        apiInfo.setName(api.getApiName());
-        apiInfo.setLcState(api.getLcState());
-        apiInfo.setTier(api.getApiTier());
-        apiInfo.setVersion(api.getApiVersion());
-        apiInfo.setProvider(api.getApiProvider());
+        apiInfo.setApiUUID(api.getAPIConfig().getUuid());
```

Comment:
Extends on BasicAPIInfo

Reply:
We had to remove tier as the BasicAPIInfo class already contains the policy. Since this is just used for troubleshooting I guess it is fine.